lib/accesslists/hierarchy.go

```diff
@@ -684,6 +684,32 @@ func collectAncestors(ctx context.Context, accessList *accesslist.AccessList, ki
 
 // GetInheritedGrants returns the combined Grants for an Access List's members, inherited from any ancestor lists.
 func GetInheritedGrants(ctx context.Context, accessList *accesslist.AccessList, g AccessListAndMembersGetter) (*accesslist.Grants, error) {
+	return getInheritedGrants(ctx, accessList, g, getInheritedGrantsOptions{
+		withMemberGrants: true,
+		withOwnerGrants:  true,
```

Comment:
The Comments says "combined Grants for an Access List's members," 

> // GetInheritedGrants returns the combined Grants for an Access List's members, inherited from any ancestor list


The result  of GetInheritedGrants function is  assigned to [AccessListResponse.InheritedMemberGrants ](https://github.com/gravitational/teleport.e/blob/master/lib/web/accesslist.go#L100)

@kiosion 
This doesn’t seem to be correct, right?
`ui.InheritedMemberGrants` should contain only the grants inherited by members.

Reply:
Since this comment is not related to this PR I have created a separate issue to discuss this behavior: 
https://github.com/gravitational/teleport.e/issues/5411